src/OAuth2User.ts

```diff
@@ -84,8 +98,11 @@ export class OAuth2User implements OAuthClient {
         const token = processTokenResponse(data);
         this.token = token;
         resolve({token});
+        this._tokenEvents.emit("tokens", this.token);
```

Comment:
should we also add a verb here? (like in tokenRefresh, accountChange, ...)

Reply:
resolved